### PR TITLE
Miw 1 july 2023

### PR DESCRIPTION
**Pull request by DominikPinsel**
This PR contain all changes from https://github.com/catenax-ng/tx-managed-identity-wallets since the 1st of July.

### CODE REVIEW

**Comment by mknoopvw** [comment]
Adds all increments and progress made since the initial PR has been issued.

No reason not to merge from my point of view.

Yet I do not know what the Eclipse Foundation's point of view is.

**Comment by carslen** [comment]
Please add a committer as a reviewer as long as you do not have a committer in your dev team if you want to get your stuff merged. :)

File: DEPENDENCIES
Comment (carslen) [inline DEPENDENCIES:1]: I'm a bit confused about this `DEPENDECIES` file. If I check your the branch from your fork (where this PR comes from) and run the Eclipse DASH tool using gradle, a different `DEPENDENCIES` file is the result. Please update the file.

**Review by carslen — CHANGES_REQUESTED** [review]
Please update `DEPENDENCIES` file, please refer to my comment.

**Comment by mknoopvw** [comment]
@DominikPinsel how much will have to be adjusted to make this PR ready to be merged?

**Comment by DominikPinsel** [comment]
> @DominikPinsel how much will have to be adjusted to make this PR ready to be merged?

I assume not much :)

File: DEPENDENCIES
Comment (DominikPinsel) [reply DEPENDENCIES:1]: This is really strange. I update the DEPENDENCIES file using a script.

```sh
#!/bin/bash

# download the latest version of the Eclipse Dash License tool
curl --output org.eclipse.dash.licenses.jar \
    https://repo.eclipse.org/service/local/repositories/dash-licenses-snapshots/content/org/eclipse/dash/org.eclipse.dash.licenses/1.0.3-SNAPSHOT/org.eclipse.dash.licenses-1.0.3-20230714.055026-52.jar


# update DEPENDENCIES file
./gradlew dependencies | grep -Poh "(?<=\s)[\w\.-]+:[\w\.-]+:[^:\s]+" | grep -v "^org\.eclipse" | sort | uniq |
    java -jar org.eclipse.dash.licenses.jar -summary DEPENDENCIES - |
    grep restricted
```

I just run the script again and got the same dependencies file :confused:

**Review by carslen — APPROVED** [review]

**Pull request merged**